.github/workflows/lint.yaml

```diff
@@ -12,9 +12,10 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: '20.11.x'
+          node-version: '20.12.x'
       - run: npm ci
       - run: npm run build-glean
       - run: cp .env-dist .env
       # Mirror old linter from CircleCI, verifies that linter succeeds
       - run: npm run lint
+      - run: node scripts/check-node-version-alignment.js
```

Comment:
If you want to use typescript for small one-off scripts like this, you could consider the `tsx` package. Or if you want to avoid dependencies, something like:

```sh
tsc --outDir dist/scripts scripts/check-node-version-alignment.ts && node dist/scripts/check-node-version-alignment.js
```

I'm assuming you don't want to have separate build/run steps in this instance.

Reply:
Yeah, one advantage of adding it in https://github.com/mozilla/blurts-server/pull/4400 is that the barrier to using it isn't that high when it's already in our dependencies. In this case I don't think it would add that much though, since we'd mostly be manually adding types for things that are only checked at runtime (because JSON, YAML, TOML and Docker files don't have type definitions).